.github/workflows/packages-build-wpk.yml

```diff
@@ -105,7 +105,7 @@ jobs:
         if: (matrix.os == 'linux' && github.event.inputs.linux_reference != '') || (matrix.os == 'windows' && github.event.inputs.windows_reference != '') || (matrix.os == 'macos' && github.event.inputs.macos_reference != '')
         run: |
           if [ "${{ github.event.inputs.docker_image_tag }}" == "auto" ]; then echo "TAG=${{ env.VERSION }}" >> $GITHUB_ENV;
-          elif [ "${{ github.event.inputs.docker_image_tag }}" == "developer" ]; then echo "TAG=${{ github.ref_name }}" >> $GITHUB_ENV;
+          elif [ "${{ github.event.inputs.docker_image_tag }}" == "developer" ]; then echo "TAG=$(sed 's|[/\]|--|g' <<< ${{ github.ref_name }})" >> $GITHUB_ENV;
```

Comment:
Could we change the `github.event.inputs.docker_image_tag` context to `inputs.docker_image_tag` to keep consistency between files? I mean, remove the `github.event` references.

Reply:
Yes, sure. I also change all the cases.